static/css/styles.css

```diff
@@ -0,0 +1,11 @@
+.content-info {
```

Comment:
Add visible styles (e.g. change the background color) here and update the screenshot to indicate that static files are served correctly

Reply:
Done! <img width="1440" alt="Screenshot 2024-09-17 at 3 18 52 PM" src="https://github.com/user-attachments/assets/715cb59f-4dcb-42a9-b6e3-83a991afe219">